api/src/main/java/graphql/nadel/schema/NadelDirectives.java

```diff
@@ -51,8 +51,15 @@
     static final DirectiveDefinition RENAMED_DIRECTIVE_DEFINITION;
     static final DirectiveDefinition HYDRATED_DIRECTIVE_DEFINITION;
     static final InputObjectTypeDefinition NADEL_HYDRATION_ARGUMENT_DEFINITION;
+    static final DirectiveDefinition DYNAMIC_SERVICE_DIRECTIVE_DEFINITION;
```

Comment:
The name of the directive is completely open to suggestions :-) 
Some names I thought:
- `dynamicService`
- `dynamic`
- `dynamicServiceResolution`

Reply:
I like `dynamicServiceResolution`

```
   node(id : ID) : Node @dynamicServiceResolution
```

Reply:
This is not a directive lots of people will use - we are likely to have 1 in the AGG

Reply:
> This is not a directive lots of people will use - we are likely to have 1 in the AGG

Yeah, I can't think of many use cases for it apart from supporting Relay